src/Components/Checkbox.tsx

```diff
@@ -1,15 +1,16 @@
 import styled from 'styled-components'
 
 interface CheckboxProps {
-  label: string
+  label: string | React.ReactNode
```

Comment:
Is React.ReactNode used when the element is a child? I don't fully understand when to use JSX.Element and when to use React.ReactNode.

Reply:
Looking at this a bit more closely it should probably just be `label: string | React.ReactElement`, as string is a valid ReactNode that type is in the list twice.